patterns/project-description.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Title: Project Description
+ * Slug: twentytwentyfour/project-description
+ * Categories: portfolio, media
+ */
+?>
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"6rem","right":"6rem","bottom":"6rem","left":"6rem"},"margin":{"top":"0px","bottom":"0px"}}},"backgroundColor":"accent-2","layout":{"type":"constrained"}} -->
+<div class="wp-block-group has-accent-2-background-color has-background" style="margin-top:0px;margin-bottom:0px;padding-top:6rem;padding-right:6rem;padding-bottom:6rem;padding-left:6rem"><!-- wp:columns {"align":"wide"} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"width":"40%"} -->
+<div class="wp-block-column" style="flex-basis:40%"><<!-- wp:heading {"textAlign":"left","fontSize":"small","fontFamily":"system-font"} -->
+<h2 class="wp-block-heading has-text-align-left has-system-font-font-family has-small-font-size"><?php echo esc_attr_x( 'Piva Hotel - Overview', 'Sample content for project title', 'twentytwentyfour' ); ?></h2>
+<!-- /wp:heading --></div> 
+<!-- /wp:column -->
+
+<!-- wp:column {"width":"9%"} -->
+<div class="wp-block-column" style="flex-basis:9%"></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"width":"60%"} -->
+<div class="wp-block-column" style="flex-basis:60%"><!-- wp:paragraph {"style":{"layout":{"selfStretch":"fixed","flexSize":"50%"},"typography":{"fontSize":"2.38rem","lineHeight":"1.3","fontStyle":"normal"}},"fontFamily":"cardo"} -->
+<p class="has-cardo-font-family" style="font-size:2.38rem;font-style:normal;line-height:1.3"><?php echo esc_attr_x( 'Photo session for the Piva Arts & Crafts Hotel near Lake Piva, Montenegro. An artificial lake was built in 1975 by dividing the Piva Kanyon to build a dam Mratinje.', 'sample content for project description', 'twentytwentyfour' ); ?></p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns -->
+
+<!-- wp:spacer {"height":"80px"} -->
+<div style="height:80px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:image {"align":"wide","id":47365,"sizeSlug":"large","linkDestination":"none","style":{"layout":{"selfStretch":"fill","flexSize":null},"border":{"radius":"16px"}}} -->
+<figure class="wp-block-image alignwide size-large has-custom-border"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/tt4_project_description.jpg" alt="<?php echo esc_attr_x( 'Photo of a kayak in the middle of a blue-green lake', 'alt text for sample image', 'twentytwentyfour' ); ?>" class="wp-image-47365" style="border-radius:16px"/></figure>
```

Comment:
Let's make the rounded borders in the markup of the block, not the actual image

Reply:
> Let's make the rounded borders in the markup of the block, not the actual image

Shouldn't we use global styles (theme.json image border radius) for a more holistic approach?